lib/rtc_video_view.dart

```diff
@@ -6,19 +6,51 @@ import 'media_stream.dart';
 import 'utils.dart';
 import 'enums.dart';
 
-class RTCVideoRenderer {
+@immutable
+class RTCVideoValue {
+  static const RTCVideoValue empty = RTCVideoValue();
+
+  final double width;
+  final double height;
+  final int rotation;
+
+  const RTCVideoValue({
+    this.width = 0.0,
+    this.height = 0.0,
+    this.rotation = 0,
+  });
+
+  double get aspectRatio {
+    if (width == 0.0 || height == 0.0) {
+      return 1.0;
+    } else {
```

Comment:
The `else` is unnecessary here since there's already a return statement in the `if` above it.